contrib/multivariate-tada/genunivdata.py

```diff
@@ -0,0 +1,50 @@
+import pandas as pd
+import torch
+
+#Simulates univariate data based on demo mutation rates in original TADA code
+
+def genUnivData(gamma, q, N_family, N_case, N_control, mut_1, mut_2):
+    #Rates for poisson distributions that the counts follow
+
+    X_d1_rates = 2 * mut_1 * gamma * N_family
```

Comment:
Here and throughout, let's pull out numerical constants into global constants so the reader can tell at a glance what `2` means in this context: is it ploidy or something else?